lib/recurly/subscription.php

```diff
@@ -4,7 +4,6 @@ class Recurly_Subscription extends Recurly_Resource
 {
   public function __construct($href = null, $client = null) {
     parent::__construct($href, $client);
-    $this->subscription_add_ons = array();
```

Comment:
@bhelx and I discovered that we can't remove this or it'll break the serialization for the subscription creation endpoint. it was being removed because the purchases endpoint doesn't allow an empty array. we need to get get a fix into the purchases endpoint before we can merge this.